baseline.gradle

```diff
@@ -105,6 +105,7 @@ subprojects {
           '-Xep:IntLongMath:ERROR',
           '-Xep:MissingSummary:ERROR',
           '-Xep:AnnotateFormatMethod:ERROR',
+          '-Xep:CollectionUndefinedEquality:ERROR',
```

Comment:
Enforced this check. So, new usage will report error for compile and the one who adds the code can check and suppress in the same PR.